src/System.Text.Utf8/src/System/Text/Utf8/Utf8String.CodePointEnumerable.cs

```diff
@@ -46,6 +46,18 @@ public CodePointEnumerator GetEnumerator()
                 }
             }
 
+            public CodePointReverseEnumerator GetReverseEnumerator()
+            {
+                if (_bytes != null)
```

Comment:
I have noticed this pattern in other places, where we have if(_bytes != null) use _bytes else use buffer. Should we unify these? I mean, say, at the constructor, construct a buffer from the bytes or vice-versa and deal with only one of them in the code?